### PR TITLE
ENT-2096 Add permissions and groups ADR

### PR DESCRIPTION
**Pull request by crice100**

### CODE REVIEW

File: docs/decisions/0002-permissions-and-groups.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Context
+=======
+Not every user should have the same level of access to portal designer. At a minimum, the following groups should be
+supported:
+
+- <site>_Moderators: Can manage their site
+- Global moderators: Can manage all sites
+- Administrators: Can manage everything
+
+Staff users should be automatically granted some level of access when they login (they should not see an error page).
```
Comment (staubina) [inline docs/decisions/0002-permissions-and-groups.rst]: Should we mention that these are Staff users on the LMS or Platform?

File: docs/decisions/0002-permissions-and-groups.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Decision
+========
```
Comment (staubina) [inline docs/decisions/0002-permissions-and-groups.rst]: Would it be worth it to make this a bulleted list of decisions?

File: docs/decisions/0002-permissions-and-groups.rst (hunk 1 above)
Comment (crice100) [reply docs/decisions/0002-permissions-and-groups.rst]: Done. I also added a link to the app-permissions repo.

File: docs/decisions/0002-permissions-and-groups.rst (hunk 2 above)
Comment (crice100) [reply docs/decisions/0002-permissions-and-groups.rst]: I have a slight preference for keeping the text in paragraph form, but could really go either way.

**Pull request merged**